src/units/pathfinder2remaster/player-core/ancestry/heritage-template

```diff
@@ -0,0 +1,108 @@
+unit: heritage/<ancestry>/remaster/<heritage>
+in: heritage/<ancestry>
+group: "_{Player Core}"
+name: "_{<heritage> <ancestry>}"
```

Comment:
As with ancestry, I suggest adding:
```
enabled: false
```
to make sure this is never accidentally included.